pkg/cmd/cli/cmd/set/env.go

```diff
@@ -489,11 +489,18 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 		if err != nil {
 			return err
 		}
+
+		resourceList := &kapi.List{}
+		resourceList.APIVersion = outputVersion.Version
+		resourceList.Kind = "List"
```

Comment:
@fabianofranz I am not sure if this is the correct way of achieving this. If this field is not set, I receive a list with no `Kind` field, preventing it from being parsed by commands like `oc create`